### PR TITLE
Top Node.js Monitoring Tools

### LINKED ISSUE #1065: Top Node.js Monitoring Tools
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Over the years, Node.js has become a popularly used JavaScript framework for rapid micro-service and API development. With workloads running on Node.js, it becomes crucial to have the right monitoring system for essentials metrics.
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
1.	What is Node.js Monitoring?
2.	What you should monitor
3.	Node.js Monitoring Tools
•	Appmetrics
•	Retrace
•	PM2
•	Prometheus
•	Clinic.js
•	AppSignal
•	Express Status Monitor

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #1065

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/top-node.js-monitoring-tools/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+When applications don't work as efficiently as they should, the responsibility of carrying out preventive and curative maintenance solely lies on your shoulders as a developer. Customers using your application may spend a lot of money trying to repair the applications without your guidance. It would be best if you had an upbeat monitoring solution to keep track of your application behavior.
```
Comment (LinusMuema) [inline articles/top-node.js-monitoring-tools/index.md:1]: This first statement is a bit long. You could find a way to rephrase it and shorten it.
Same as other few statements in the article. You can use Hemingway to help you

**Review by LinusMuema — CHANGES_REQUESTED** [review]
Hi @EKahuha 
This is a great article. However, your language is quite complex and it would be better to simplify it by using simpler words apart from the technical ones. 
Just check the one suggestion and I have, otherwise, everything else checks out :+1:

**Comment by EKahuha** [comment]
@LinusMuema, I have updated this article.

**Comment by LinusMuema** [comment]
Hi @hectorkambow
You could give this article a plag check and final review.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
Please be sure to go over some of minor edits I made for flow @EKahuha  
cc @LinusMuema  🚀

**Pull request merged**